Test Parametrization

kamalojasv181 opened the pull request:
Related to #2522

Description: Added pytest parameterization for tests/ignite/contrib/handlers/test_base_logget.py - test_attach function
@vfdev-5

vfdev-5 commented on tests/ignite/contrib/handlers/test_base_logger.py:
```diff
@@ -169,52 +169,52 @@ def test_opt_params_handler_on_non_torch_optimizers():
 
-def test_attach():
+@pytest.mark.parametrize(
+    "event,n_calls,kwargs",
```
```suggestion
    "event, n_calls, kwargs",
```

vfdev-5 reviewed (approved):
@kamalojasv181 Thanks for the PR!
LGTM

kamalojasv181 commented:
![image](https://user-images.githubusercontent.com/43926001/179207252-f4018dc6-054e-4f48-8105-71aaf2b2432e.png)
@vfdev-5 I do not understand. I did not even change this file. Is it possible that this problem was with the main branch itself?

vfdev-5 commented:
@kamalojasv181 yes, that's true it is an issue with master.

kamalojasv181 commented:
@vfdev-5 can you please approve the tests? I have deleted the unnecessary comments and added an ignore for the line, showing an error with the nightly PyTorch version. We can do that better by creating a separate variable instead of overriding the old one. However, that might break the code. Hence we can deal with it in a different PR. I have also skipped the mypy test for the stable PyTorch version.

vfdev-5 commented:
@kamalojasv181 thanks for the update, could you please create a separate PR for typing fixes. It's better to split the updates into different commits

kamalojasv181 commented:
@vfdev-5 I have created a separate PR for it https://github.com/pytorch/ignite/pull/2616

vfdev-5 commented on ignite/handlers/lr_finder.py:
```diff
@@ -493,7 +493,7 @@ def __init__(self, optimizer: Optimizer, start_lr: List[float], end_lr: float, n
 
         # override base_lrs
-        self.base_lrs = start_lr
+        self.base_lrs = start_lr  # type: ignore
```
@kamalojasv181 please revert this change
